src/main/java/docSharing/entities/document/Document.java

```diff
@@ -1,4 +1,20 @@
 package docSharing.entities.document;
 
-public class Document {
+import docSharing.entities.Permission;
+import docSharing.entities.User;
+import javax.persistence.*;
+import java.util.HashMap;
+import java.util.List;
+
+
+public class Document extends File{
+
+    private String content;
+
+    public Document(int id, String url, HashMap<Permission, List<User>> authorized, MetaData metaData, List<UpdateLog> updateLogs, String content) {
```

Comment:
כאן עדיף:
1. ID שיהיה auto increment, אז לא צריך לשלוח אותו
2. לא צריך לשלוח את כל המשתנים האלה, פשוט לאתחל בקונסטרקטור את כולם כריקים. 
3. הקונסטרקטור יקבל User ויכניס אותו כOWNER למפה, יאתחל אותו כ-Creator בmetadata.
4. ה-content יהיה ריק (כי רק נוצר הקובץ)

Reply:
1. ' Variable 'id' might not have been initialized '- because it's final- look also in the User class. 
2. fixed.
3. fixed.
4. fixed.